src/aind_metadata_mapper/open_ephys/utils/naming_utils.py

```diff
@@ -0,0 +1,390 @@
+""" Utils to process naming of stimulus columns"""
+
+import numpy as np
+import re
+import warnings
+import logging
+
+INT_NULL = -99
+logger = logging.getLogger(__name__)
+
+# defaults
+DEFAULT_OPTO_CONDITIONS = {
```

Comment:
maybe all these default values can be saved in another file?

Reply:
Done, moved to the constants.py